### PR TITLE
Enabled INDEXES_RENAMING param for MySQL - fixes (GH Issue #136)

### LINKED ISSUE #136: [MySQL] Handle duplicate index names
MySQL allows more than one indexes to have same name in the same database.
```
mysql> CREATE TABLE t1 (k integer, v integer);
Query OK, 0 rows affected (0.04 sec)

mysql> CREATE TABLE t2 (k integer, v integer);
Query OK, 0 rows affected (0.02 sec)

mysql> CREATE INDEX idx on t1 (v);
Query OK, 0 rows affected (0.01 sec)
Records: 0  Duplicates: 0  Warnings: 0

mysql> CREATE INDEX idx on t2 (v);
Query OK, 0 rows affected (0.02 sec)
Records: 0  Duplicates: 0  Warnings: 0

mysql> SELECT DISTINCT TABLE_NAME, INDEX_NAME FROM INFORMATION_SCHEMA.STATISTICS WHERE TABLE_SCHEMA = 'test';
+------------+------------+
| TABLE_NAME | INDEX_NAME |
+------------+------------+
| t1         | idx        |
| t2         | idx        |
+------------+------------+
2 rows in set (0.00 sec)
```
Migrating such a database will fail in the `import data` command (after data is imported and indexes are being created). e.g.
```
All the tables are imported
setting resume value for sequences           done
creating indexes           
    ERROR: relation "idx_fk_film_id" already exists (SQLSTATE 42P07)
    STATEMENT: CREATE INDEX idx_fk_film_id ON inventory (film_id);
```
https://downloads.mysql.com/docs/sakila-db.zip schema can be used to reproduce the issue.

Postgres and Oracle does not allow duplicate index names.

To handle such situations, `ora2pg` does provide an `INDEX_RENAMING` option (https://ora2pg.darold.net/documentation.html#:~:text=but%20can%20help.-,INDEXES_RENAMING,-Enable%20this%20directive).

yb_migrate should enable the `INDEX_RENAMING` option (based on user input).

Until the issue is fixed, users can edit the `<export-dir>/schema/tables/INDEXES_table.sql` file to rename the duplicate indexes and retry the `import data` command. The `import data` will straight-away proceed to creation of indexes.

### PR DESCRIPTION
**Pull request by priyanshi-yb**
Resolves #136

### CODE REVIEW

File: yb-voyager/src/srcdb/source.go [hunk 1]
```diff
@@ -189,6 +189,8 @@ func (source *Source) PopulateOra2pgConfigFile(configFilePath string) {
 			lines[i] = "PARALLEL_TABLES " + strconv.Itoa(source.NumConnections)
 		} else if strings.HasPrefix(line, "PG_VERSION") {
 			lines[i] = "PG_VERSION " + strconv.Itoa(11) //TODO YugabyteDB compatible with postgres version ?
+		} else if strings.HasPrefix(line, "INDEXES_RENAMING") && source.DBType == "mysql"{
+			lines[i] = "INDEXES_RENAMING " + strconv.Itoa(1)
```
Comment (amit-yb) [inline yb-voyager/src/srcdb/source.go:193]: Nit: why not simply write
```
lines[i] = "INDEX_RENAMING 1"
```

**Review by amit-yb — APPROVED** [review]
LGTM with a minor nit.

File: yb-voyager/src/srcdb/source.go [hunk 2]
```diff
@@ -189,6 +189,8 @@ func (source *Source) PopulateOra2pgConfigFile(configFilePath string) {
 			lines[i] = "PARALLEL_TABLES " + strconv.Itoa(source.NumConnections)
 		} else if strings.HasPrefix(line, "PG_VERSION") {
 			lines[i] = "PG_VERSION " + strconv.Itoa(11) //TODO YugabyteDB compatible with postgres version ?
```
Comment (sanyamsinghal) [inline yb-voyager/src/srcdb/source.go:191]: remove this TODO from here.

**Review by sanyamsinghal — APPROVED** [review]
LGTM. Please a documentation ticket(as discussed) before merging this and if possible complete that soon since its small addition.

**Pull request merged**